packages/experimental/src/components/UserProfileImage/UserProfileImage.js

```diff
@@ -0,0 +1,128 @@
+/**
+ * Copyright IBM Corp. 2020, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Canary } from '../_Canary';
+import { pkg } from '../../settings';
+const componentName = 'UserProfileImage';
+
+import {
+  User16,
+  User20,
+  User24,
+  User32,
+  Group16,
+  Group20,
+  Group24,
+  Group32,
+} from '@carbon/icons-react';
+
+//import { expPrefix } from '../../global/js/settings';
+import cx from 'classnames';
+
+const blockClass = `${pkg.prefix}-user-profile-avatar`;
+
+export const UserProfileImage = !pkg.isComponentEnabled(componentName)
+  ? // Return canary if not released or flag not set
+    () => <Canary component={componentName} />
+  :({
+  backgroundColor,
+  icon = 'user',
```

Comment:
We should use defaultProps to supply default values for props that aren't marked required.

e.g.
```js
UserProfileImage.defaultProps = {
  icon: 'user',
  size: 'x-large',
};
```